code/modules/events/holiday/halloween.dm

```diff
@@ -53,6 +53,43 @@
 	if(IsMultiple(activeFor, 4))
 		spawn_meteors(3, meteorsSPOOKY) //meteor list types defined in gamemode/meteor/meteors.dm
 
+//Creepy clown invasion
+/datum/round_event_control/creepy_clowns
+	name = "Clowns"
+	typepath = /datum/round_event/creepy_clowns
+	holidayID = HALLOWEEN
+	weight = 20
+	earliest_start = 0
+
+/datum/round_event/creepy_clowns
+	endWhen = 40
+
+/datum/round_event/creepy_clowns/start()
+	for(var/mob/living/carbon/human/H in living_mob_list)
+		if(!H.client || !istype(H))
+			return
+		H << "<span class='danger'>Honk...</span>"
+		H << 'sound/spookoween/scary_clown_appear.ogg'
+		new /obj/effect/hallucination/delusion(H.loc,H,force_kind="clown",duration=300,skip_nearby=0)
+
+/datum/round_event/creepy_clowns/tick()
+	if(IsMultiple(activeFor, 3))
+		for(var/mob/living/carbon/human/H in living_mob_list)
+			if (prob(66))
+				playsound(H.loc, pick('sound/spookoween/scary_horn.ogg','sound/spookoween/scary_horn2.ogg', 'sound/spookoween/scary_horn3.ogg'), 300, 1)

```

Comment:
Wouldn't you want to play the sound directly to them? Also, volume caps at 100 for playback; having it any higher has no effect.